webpack.config.js

```diff
@@ -14,13 +18,21 @@ module.exports = {
           },
         },
       },
+      {
+        test: /\.css$/,
+        loader: "style-loader"
+      },
+      {
+        test: /\.css$/,
+        loader: "css-loader",
+        options: {
+          modules: true,
+          localIdentName: "[name]__[local]___[hash:base64:5]"
```

Comment:
I didn't add this "localIdentName" to my options and css loader worked fine. Unsure if it's really necessary here.